### PR TITLE
feat: add HyperText component

### PR DESCRIPTION
**Pull request by premdasvm**
Implementation of [Hyper Text Component](https://magicui.design/docs/components/hyper-text) by Magic UI

### CODE REVIEW

File: components/content/inspira/ui/hyper-text/HyperText.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const props = defineProps({
+    text: {
+        type: String,
+        required: true,
+    },
+    duration: {
+        type: Number,
+        default: 800,
+    },
+    className: {
```
Comment (rahul-vashishtha) [inline components/content/inspira/ui/hyper-text/HyperText.vue:30]: `className` is the part of the react. You can make it class and then use $props.class with `cn` utility to apply to the element you want.
Please refer to other component

File: content/2.components/hyper-text.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Component Code
+
+You can copy and paste the following code to create this component:
+
+::CodeViewer{filename="HyperText.vue" language="vue" componentName="HyperText" type="ui" id="hyper-text"}
+::
+
+## Credits
+
+-   Inspired by [Magic UI's Hyper Text](https://magicui.design/docs/components/hyper-text) component.
```
Comment (rahul-vashishtha) [inline content/2.components/hyper-text.md:27]: Add your name with profile link in credits section.

**Review by rahul-vashishtha — CHANGES_REQUESTED** [review]

File: components/content/inspira/ui/hyper-text/HyperText.vue (hunk 1 above)
Comment (premdasvm) [reply components/content/inspira/ui/hyper-text/HyperText.vue]: Done.

File: content/2.components/hyper-text.md (hunk 2 above)
Comment (premdasvm) [reply content/2.components/hyper-text.md:27]: Done.

**Review by rahul-vashishtha — APPROVED** [review]

**Pull request merged**